static/src/js/containers/SidebarContainer/Facets/FacetsItem.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { uniqueId } from 'lodash'
+
+import { generateFacetArgs } from '../../../util/facets'
+
+class FacetsItem extends Component {
+  constructor(props) {
+    super(props)
+
+    const { facet } = props
+
+    this.state = {
+      applied: facet.applied
+    }
+
+    this.onFacetChange = this.onFacetChange.bind(this)
+  }
+
+  onFacetChange(changeHandlerArgs, e) {
+    const { changeHandler } = this.props
+    const { applied } = this.state
+
+    this.setState({
+      applied: !applied
+    })
+
+    changeHandler(e, changeHandlerArgs)
```

Comment:
Generally I've seen these named `onChangeHandler` is that a convention or am I looking too deep into tutorials?

Reply:
You are correct. This is an odd case where it is being passed down as a property on an object, so its not the typical naming convention.